packages/eslint-plugin/package.json

```diff
@@ -30,29 +30,29 @@
     "typescript": "^3.9.2"
   },
   "peerDependencies": {
-    "eslint": "^6.0.0 || ^7.0.0"
+    "eslint": "^7.0.0"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.1.0",
-    "@typescript-eslint/parser": "^4.1.0",
+    "@typescript-eslint/eslint-plugin": "^4.15.0",
+    "@typescript-eslint/parser": "^4.15.0",
     "babel-eslint": "10.1.0",
-    "change-case": "^4.1.1",
+    "change-case": "^4.1.2",
     "common-tags": "^1.8.0",
-    "eslint-config-prettier": "6.14.0",
+    "eslint-config-prettier": "7.2.0",
     "eslint-module-utils": "2.6.0",
     "eslint-plugin-babel": "5.3.1",
     "eslint-plugin-eslint-comments": "3.2.0",
     "eslint-plugin-graphql": "4.0.0",
     "eslint-plugin-import": "2.22.1",
-    "eslint-plugin-jest": "24.1.0",
-    "eslint-plugin-jest-formatting": "2.0.0",
+    "eslint-plugin-jest": "24.1.3",
+    "eslint-plugin-jest-formatting": "2.0.1",
     "eslint-plugin-jsx-a11y": "6.4.1",
     "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-prettier": "3.3.0",
-    "eslint-plugin-promise": "4.2.1",
-    "eslint-plugin-react": "7.21.5",
+    "eslint-plugin-prettier": "3.3.1",
+    "eslint-plugin-promise": "4.3.1",
+    "eslint-plugin-react": "7.22.0",
```

Comment:
Potential rules/tweaks:
* [`jsx-no-constructed-context-values`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-constructed-context-values.md)
* [`jsx-newline`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-newline.md)
* jsx-key added [`checkKeyMustBeforeSpread`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md#checkkeymustbeforespread-default-false) option

Reply:
`jsx-newline` will cause a lot of churn without much value. I disabled it.
I enabled the other two.